magento/magento2
# Remove Great Britain from EU country list

### Description (*)
Remove Great Britain from EU country list

### Related Pull Requests


### Fixed Issues (if relevant)
1. Fixes magento/magento2#31506

### Manual testing scenarios (*)
1. Go to European Union Countries configuration and check if United Kingdom is selected

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#31506: Remove Great Britain from EU country list
